crates/testing/tests/tests_5/combined_network.rs

```diff
@@ -72,7 +70,7 @@ async fn test_combined_network_cdn_crash() {
         // allow more time to pass in CI
         completion_task_description: CompletionTaskDescription::TimeBasedCompletionTaskBuilder(
             TimeBasedCompletionTaskDescription {
-                duration: Duration::from_secs(120),
+                duration: Duration::from_secs(220),
```

Comment:
How come we upped the time here?

Reply:
This test was failing because it was taking to long on this branch and main so I increased the time to make it less likely to fail